tests/phpunit/tests/test-admin-filters-term.php

```diff
@@ -730,4 +730,38 @@ public function test_update_term_from_admin() {
 
 		unset( $_REQUEST, $_POST );
 	}
+
+	public function test_update_term_slug() {
+		$original_name = 'Test Me';
+		$new_slug      = 'well-tested';
+
+		$cat_en = $this->factory()->category->create_and_get(
+			array(
+				'name' => $original_name,
+			)
+		);
+		self::$model->term->set_language( $cat_en->term_id, 'en' );
+
+		$this->assertSame( sanitize_title( $original_name ), $cat_en->slug, 'The category slug is well created.' );
```

Comment:
I'm not sure about this error message x) (if it's "well created", then what is the problem?).
#CopyPasteDrama

Reply:
Yup! Done in 71acd76